package.json

```diff
@@ -18,10 +18,12 @@
 		"url": "https://github.com/BranchMetrics/Smart-App-Banner-Deep-Linking-Web-SDK/issues"
 	},
 	"homepage": "https://github.com/BranchMetrics/Smart-App-Banner-Deep-Linking-Web-SDK",
+	"peerDependencies": {
```

Comment:
CircleCI needs grunt as a peer dependency, else it will throw errors during the tests